src/styles/main.scss

```diff
@@ -1,3 +1,17 @@
+@import url("../styles/header.scss");
+@import url("../styles/nav.scss");
+@import url("../styles/products.scss");
+@import url("../styles/product.scss");
+@import url("../styles/stars.scss");
+
+html {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+  max-width: 1200px;
```

Comment:
it could be better to set width at other place, your header doesn't stretch to the full width